src/helpers/tracks-file-parser/worker/tracks-file-parser-worker.ts

```diff
@@ -1,14 +1,14 @@
 /// <reference lib='WebWorker' />
 
-import { parse as parseMetadata } from 'id3-parser/src/index'
+import { Buffer } from 'buffer'
+import { parseBuffer as parseMetadata } from 'music-metadata'
```

Comment:
You could possibly use import [`music-metadata-browser`](https://github.com/Borewit/music-metadata-browser) and pass [`Response.blob()`](https://developer.mozilla.org/en-US/docs/Web/API/Response/blob) to [`parseblob()`](https://github.com/Borewit/music-metadata-browser#parseblob-function)